docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,112 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
+  sensible warning
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+- [ ] Select a Windows image, and expect a sensible warning
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+- [ ] Remove all drives
+- [ ] Switch drive selection
+- [ ] Insert a single drive, and expect auto-selection
+- [ ] Insert one system drive, and don't expect auto-selection
```

Comment:
Was the "Insert one system drive" here supposed to say "Insert another drive"?

Reply:
Clarified!

Reply:
Hmm? Still says "Insert one system drive, and don't expect auto-selection" ??

Reply:
Is that really incorrect? We don't do auto-selection for system drives

Reply:
But how exactly do you "insert one system drive"??

Perhaps I'm getting confused with what you mean, and there should also be a separate checkpoint which says "Insert two drives, and don't expect auto-selection"?

Reply:
Oops, sorry; I didn't push my local changes :)